src/content/eduhelx/eduHelxReadMore.mdx

```diff
@@ -6,7 +6,7 @@ import photoWomanWithComputer from "../../images/eduhelx/photo-woman-with-comput
 
 <MaxWidthWrapper gutterBottom>
   
-<HeadingSection wrap={false}>
+<HeadingSection wrap={false} maxWidth="lg">
```

Comment:
we can remove the `wrap` prop here as well as from the `/eduhelx/eduHelxReadMore.mdx` section